pages/blog/mistake-odyssey.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Mistake Odyssey"
+Subtitle: "Git, Github, VS Code and Markdown for non-programmers"
+date: 2020-07-08T06:00:00+01:00
+type: Communication
+tags:
+  - Git
+  - GitHub
+  - VS Code
+  - Markdown
+cover: img/mistake odyssey-Carolinie-cavalli.webp
```

Comment:
Try to always avoid spaces in file names. My recommendation is that you rename the file and then come back to this line and change it. Also, using lower case is a good habit when working with URLs (this file name will become a URL when used inside HTML `<img>` tag).

Reply:
Thanks, I will tried :)